js/src/wp-seo-featured-image.js

```diff
@@ -160,6 +161,30 @@ import a11ySpeak from "a11y-speak";
 		if ( "undefined" !== typeof $featuredImageElement.prop( "src" ) ) {
 			featuredImagePlugin.setFeaturedImage( $( "#set-post-thumbnail " ).html() );
 		}
+
+		// Fallback for Gutenberg, as the featured image id does not exist there.
+		let imageData;
+		if ( isGutenbergDataAvailable() ) {
+			wp.data.subscribe( () => {
+				const featuredImageId = wp.data.select( "core/editor" ).getEditedPostAttribute( "featured_media" );
+
+				if ( featuredImageId === undefined ) {
```

Comment:
If `featuredImageId` is `null`, this check won't work. Consider also catching the situation where the image ID is potentially `null`.